src/containers/categories-title-input/CategoriesTitleInput.jsx

```diff
@@ -0,0 +1,160 @@
+import { useState } from 'react'
+import { Box, InputAdornment, Typography } from '@mui/material'
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import AppButton from '~/components/app-button/AppButton'
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+import SearchIcon from '@mui/icons-material/Search'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { authRoutes } from '~/router/constants/authRoutes'
+import { styles } from '~/containers/categories-title-input/CategoriesTitleInput.styles'
+import { useSearchParams } from 'react-router-dom'
```

Comment:
On 8th line we have import from 'react-router-dom', so combine this two imports. 
```suggestion
import { useNavigate, useSearchParams } from 'react-router-dom'
```

Reply:
I combined these two imports.
Thank you!